lib/alternate-login-prompt/style.scss

```diff
@@ -0,0 +1,77 @@
+.alternate-login__overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: rgba($studio-gray-20, 0.75);
+}
+
+.alternate-login__dismiss {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  line-height: 60px;
+  margin: 0 16px;
+  width: 100%;
+
+  svg {
+    height: 16px;
+    width: 16px;
+    margin: auto 0;
+  }
+}
+
+.alternate-login__modal {
+  align-items: center;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 20%;
+  max-width: 420px;
+  padding: 0 24px 16px;
+  font-size: 16px;
+
+  p {
+    margin: 0 26px;
+    margin-block-start: 0;
+  }
+
+  p:not(:last-of-type) {
+    padding-bottom: 20px;
+  }
+
+  .alternate-login__button-row {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+    width: 100%;
+
+    a {
+      padding-right: 12px;
+    }
+    .button-borderless {
+      color: $studio-simplenote-blue-50;
+    }
+  }
+
+  &:focus {
+    outline: 0;
```

Comment:
One additional note — what do you think about not setting `outline: 0`? Keyboard users rely upon focus styles to track focus selection in the UI. Alternatively, we could design our own focus style that does not rely upon the default `outline` styling. 

I also posed this question in https://github.com/Automattic/simplenote-electron/pull/2685#discussion_r590577004.

Reply:
This seems out of scope for this PR. If we choose to remove outlines or design our own focus style, we can update this along with all the other extant dialogs.